pkg/controller/dataimportcron-controller.go

```diff
@@ -447,7 +457,7 @@ func (r *DataImportCronReconciler) cleanup(ctx context.Context, dataImportCron *
 	if err := r.client.Update(ctx, dataImportCron); err != nil {
 		return err
 	}
-	return nil
+	return r.checkFailingDataImportCrons()
```

Comment:
why do we care about it in cleanup?

Reply:
Cleanup means we need to update the metric since the "failing" cron got deleted
Otherwise we keep firing the alert even though there is not failing cron anymore